src/qt/bitcoin.cpp

```diff
@@ -702,9 +702,20 @@ int main(int argc, char *argv[])
     // from the registry (Windows) or a configuration file (Linux/OSX)
     // we need to check to see if we need to migrate old settings to the new location
 #ifdef BITCOIN_CASH
-    // TODO: Add migration code which first tries to move from BU to BUcash settings,
-    //       or if there weren't pre-existing BU settings, try to migrate from a legacy settings
+    bool bMigrated = false;
+    // For BUCash, first try to migrate from BTC BU settings
```

Comment:
nit, although i don't think its formally specified anywhere, in bitcoin we use an f prefix instead of b to indicate a bool.